llvm/tools/llc/NewPMDriver.cpp

```diff
@@ -121,10 +121,10 @@ int llvm::compileModuleWithNewPM(
   registerCodeGenCallback(PIC, LLVMTM);
 
   LoopAnalysisManager LAM;
+  MachineFunctionAnalysisManager MFAM;
```

Comment:
ultra nit, I'd put this before `LoopAnalysisManager` since MachineFunction is further from Function than Loop is from Function